MiniApp/Classes/core/Extensions/NSError+MiniApp.swift

```diff
@@ -107,5 +107,6 @@ enum MiniAppSDKErrorCode: Int {
          adNotLoaded,
          adNotDisplayed,
          miniAppNotFound,
-         metaDataFailure
+         metaDataFailure,
+         failedToConformToProtocol
```

Comment:
This seemed a good place to have this generic error. We should discuss about the failedToConformToProtocol error, we have it in many places but not in a more generic error.